src/LiberisLabs.CompaniesHouse.IntegrationTests/Tests/CompanyFilingHistoryTests/CompanyFilingHistoryTestsInvalid.cs

```diff
@@ -0,0 +1,27 @@
+﻿using NUnit.Framework;
+
+namespace LiberisLabs.CompaniesHouse.IntegrationTests.Tests.CompanyFilingHistoryTests
+{
+    [TestFixture]
+    public class CompanyFilingHistoryTestsInvalid : CompanyFilingHistoryTestBase
+    {
+        private const string InvalidCompanyNumber = "ABC00000";
+
+        [SetUp]
+        protected override void When()
+        {
+            WhenRetrievingAnInvalidCompanyProfile();
+        }
+
+        [Test]
+        public void ThenTheProfileIsNotReturned()
+        {
+            Assert.That(_result.Data.Items, Is.Null);
```

Comment:
Ah right i see, I thought we was explicitly setting it to null within `CompaniesHouseCompanyFilingHistoryClient` when it was a 404 but this is actually the response from the web service? Lets leave it how it is and I'll merge it in.